kuma/search/jinja2/search/results.html

```diff
@@ -22,15 +22,16 @@
 {% block bodyclass %}search-results search-results-no-demo search-navigator-flush{% endblock %}
 
 {% macro doc_block(doc, index) -%}
+  {% set doc_url = wiki_url(doc.slug) if wiki else doc.url %}
```

Comment:
```suggestion
  {% set doc_url = wiki_url(doc.slug) %}
```
I don't think it's ever NOT `wiki`. The React site (aka. the not-wiki) uses `search-results-page.jsx` (or whatever it's called). 
I still like the idea making a variable once.